tests/components/zha/test_config_flow.py

```diff
@@ -237,7 +237,7 @@ async def test_discovery_via_usb(detect_mock, hass):
         await hass.async_block_till_done()
 
     assert result2["type"] == FlowResultType.CREATE_ENTRY
-    assert "zigbee radio" in result2["title"]
+    assert result2["title"] == "zigbee radio"
```

Comment:
Just a question... as this is a title should we Capitalize it?

Reply:
This would typically be a product name, I don't think we should capitalize it but leave it as the manufacturer has defined it.